projects/packages/connection/src/class-rest-connector.php

```diff
@@ -256,6 +256,17 @@ public function __construct( Manager $connection ) {
 				),
 			)
 		);
+
+		// Set the connection owner.
```

Comment:
I believe this endpoint might be better suited for Jetpack-the-plugin. What do you think?